gremlin-driver/src/main/java/org/apache/tinkerpop/gremlin/driver/Cluster.java

```diff
@@ -986,13 +996,26 @@ public Builder port(final int port) {
          * handshake and SSL handshake. Beyond this duration an exception would be thrown.
          *
          * Note that this value should be greater that SSL handshake timeout defined in
-         * {@link io.netty.handler.ssl.SslHandler} since WebSocket handshake include SSL handshake.
+         * {@link io.netty.handler.ssl.SslHandler} since WebSocket handshake include SSL handshake. This value should be
+         * less than {@link Builder#maxWaitForConnection}.
```

Comment:
please update comment to include `connectionSetupTimeoutMillis`

Reply:
Could you clarify this? Did you mean adding a param line? If so, I didn't do this as it seemed a bit redundant and it isn't done for any of the other params in this class.

Reply:
I am sorry sorry, copy/paste attempt resulted in wrong comment. 

What I wanted to say was to update the java doc comment since "Note that this value should be greater that SSL handshake timeout defined in SslHandler" is not true anymore. We should replace this with information with the new timeout we are introducing `sslHandshakeTimeoutMillis`